test/http-test.js

```diff
@@ -12,9 +12,6 @@ const MTX = require('../lib/primitives/mtx');
 const FullNode = require('../lib/node/fullnode');
 const pkg = require('../lib/pkg');
 
-if (process.browser)
-  return;
-
```

Comment:
Any reason to move this inside the test case? Probably better to exit earl.

Reply:
I moved it inside of the test case to match another place in the codebase.
https://github.com/bcoin-org/bcoin/blob/01a71561b1677981a4b355f7deec46f50e68a855/test/node-test.js#L98-L102

I'm not the hugest fan of the free floating code, I do understand your point of wanting to exit at the earliest point possible. Is there a reason as to why most of the initial state for the test happens outside of the `Describe` blocks?

Reply:
Reverted this change